src/server/server_family.h

```diff
@@ -350,6 +360,8 @@ class ServerFamily {
 
   mutable util::fb2::Mutex peak_stats_mu_;
   mutable PeakStats peak_stats_;
+
+  LoadingStats loading_stats_;
```

Comment:
What's guarding it? 🙂 I assume we can access it in parallel from GetMetrics() and update it form the cron snapshot job

Reply:
My mistake :cry: . Fixed it in the last commit